Framework/src/ObjectsManager.cxx

```diff
@@ -124,4 +127,28 @@ int ObjectsManager::getNumberPublishedObjects()
   return mMonitorObjects->GetLast() + 1; // GetLast returns the index
 }
 
+void ObjectsManager::setDefaultDrawOptions(const std::string& objectName, const std::string& options)
+{
+  MonitorObject* mo = getMonitorObject(objectName);
+  mo->addOrUpdateMetadata(gDrawOptionsKey, options);
+}
+
+void ObjectsManager::setDefaultDrawOptions(TObject* obj, const std::string& options)
+{
+  MonitorObject* mo = getMonitorObject(obj->GetName());
+  mo->addOrUpdateMetadata(gDrawOptionsKey, options);
+}
+
+void ObjectsManager::setDisplayHint(const std::string& objectName, const std::string& hints)
+{
+  MonitorObject* mo = getMonitorObject(objectName);
+  mo->addOrUpdateMetadata(gDisplayHintsKey, hints);
+}
+
+void ObjectsManager::setDisplayHint(TObject* obj, const std::string& hints)
+{
+  MonitorObject* mo = getMonitorObject(obj->GetName());
```

Comment:
This is unrelated to this PR, but maybe we should consider protecting those MOs from being deleted by returning a const pointer (with non-const object pointed to).

Reply:
Agreed, could you create a JIRA ticket ?

Reply:
https://alice.its.cern.ch/jira/browse/QC-361